### PR TITLE
add new run service account target

### PR DESCRIPTION
**Pull request by ciaranRoche**
## Overview

Jira: https://issues.redhat.com/browse/INTLY-4503

## Verification

- Clone this branch
- Run `make cluster/prepare`
- Seed the cluster with some resources 
- Run `make code/run/service_account`
- Ensure the operator behaves as expected

### CODE REVIEW

File: deploy/role.yaml [hunk 1]
```diff
@@ -88,4 +88,96 @@ rules:
   verbs:
   - '*'
   resources:
-  - credentialsrequests
\ No newline at end of file
+  - credentialsrequests
+---
+apiVersion: rbac.authorization.k8s.io/v1
+kind: ClusterRole
```
Comment (dimitraz) [inline deploy/role.yaml]: We have a separate file for cluster roles - https://github.com/integr8ly/cloud-resource-operator/blob/master/deploy/cluster_role.yaml

File: deploy/role_binding.yaml [hunk 2]
```diff
@@ -9,3 +9,16 @@ roleRef:
   kind: Role
   name: cloud-resource-operator
   apiGroup: rbac.authorization.k8s.io
+---
+kind: ClusterRoleBinding
```
Comment (dimitraz) [inline deploy/role_binding.yaml]: I would be in favour of moving this out to a separate file

File: Makefile [hunk 3]
```diff
@@ -26,6 +26,18 @@ build:
 run:
 	RECTIME=30 operator-sdk up local --namespace=""
 
+.PHONY: setup/service_account
+setup/service_account:
```
Comment (austincunningham) [inline Makefile]: Should we have a make target to remove the service account as well ? Did have issues with this in the passed with integreatly's service account locking me out. 

Wait its tied to the namespace, so as soon as the namespace is gone it should be gone right ?

Comment (ciaranRoche) [reply Makefile]: Yeah it is all added to the `namespace`, just have to remember to log yourself back in after running this target and everything should be good

**Pull request merged**